native_client/BUILD

```diff
@@ -85,14 +77,12 @@ tf_cc_shared_object(
     linkopts = select({
         "//tensorflow:darwin": [],
         "//tensorflow:linux_x86_64": LINUX_LINKOPTS,
-        "//tensorflow:rpi3": LINUX_LINKOPTS + ["-l:libstdc++.a"],
-        "//tensorflow:rpi3-armv8": LINUX_LINKOPTS + ["-l:libstdc++.a"],
         "//tensorflow:windows": [],
         "//conditions:default": []
     }) + tflite_linkopts(),
     deps = select({
         "//native_client:tflite": [
-            "//tensorflow/contrib/lite/kernels:builtin_ops",
+            "//tensorflow/lite/kernels:builtin_ops",
```

Comment:
Please ignore this change, it's for local testing.

Reply:
Yes, when I rebase this PR on top of the r1.13 changes all of this will be gone.